app/assets/javascripts/oxalis/geometries/arbitrary_plane.js

```diff
@@ -100,6 +112,13 @@ class ArbitraryPlane {
           matrix[15],
         );
 
+        mesh.matrix.identity();
+
+        // const [xFactor, yFactor] = applyAspectRatioToWidth(this.aspectRatio, 1);
```

Comment:
Not sure whether you want to use the commented out code in the future, otherwise this can probably be reverted to the way it was before.